src/modules/tracker.js

```diff
@@ -2235,6 +2235,391 @@ class Tracker {
     return new Error('parameters are required of type object');
   }
 
+  /**
+   * Send ASA request submitted event
+   *
+   * @function trackAssistantSubmit
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {string} parameters.intent - Intent of of user request
+   * @param {string} [parameters.section] - The section name for the item Ex. "Products"
```

Comment:
Why do we have `[parameters.section="Products"]` in some functions but `[parameters.section]` in others? Is it intentional?